src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+
 public class SalaryInfo {
+    private static final int FIRST_ARRAY_ELEMENT = 0;
+    private static final int SECOND_ARRAY_ELEMENT = 1;
+    private static final int THIRD_ARRAY_ELEMENT = 2;
+    private static final int FOURTH_ARRAY_ELEMENT = 3;
+    private static final int ONE_DAY = 1;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder sb = new StringBuilder("Report for period " + dateFrom + " - " + dateTo);
```

Comment:
Same about context
```suggestion
        StringBuilder result = new StringBuilder("Report for period " + dateFrom + " - " + dateTo);
```
String builder - is not a service class. It is just an improved version of the string.
Also it is a bad practice to use abbreviations.